### ISSUE TITLE
False negative on Layout/MultilineMethodCallIndentation for safe navigation

### ISSUE THREAD

**Issue by brkn** [rubocop/rubocop#12199]
`Layout/MultilineMethodCallIndentation` doesn't catch safe navigation chains.

## Current behaviour

This is considered correctly aligned.

```rb
# frozen_string_literal: true

author.books
      .first
    &.tags
        &.map { |tag| "tag: #{tag}" }
```

> 1 file inspected, no offenses detected

## Describe the solution you'd like

It should correctly align multiline chains whether it has safe navigation or not.

## Additional context

```sh
> rubocop -v
1.56.2
```
